src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System.Runtime.InteropServices.JavaScript.Tests.csproj

```diff
@@ -12,6 +12,7 @@
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
     <!-- this unit test depends on JavaScript interop and doesn't need heavy threading. So we can run it in UI thread. TODO: Drop this when we have automatic JSInterop dispatch to UI thread. -->
     <_XUnitBackgroundExec>false</_XUnitBackgroundExec>
+    <WasmNativeStrip>false</WasmNativeStrip>
```

Comment:
Add a comment explaining why this is here if possible

Reply:
removed